src/views/UserSettings.vue

```diff
@@ -1,18 +1,43 @@
 <template>
+<div v-if="countriesData">
   <h1 class="mb-1">
-    Settings
-  </h1>
+    {{ $t('UserSettings.Title') }} </h1>
 
   <v-form @submit.prevent="updateSettings">
     <v-row>
       <v-col cols="12" sm="6">
-        <v-card title="Adding prices" prepend-icon="mdi-tag-plus-outline">
+        <v-card :title="$t('UserSettings.ChangeLanguage')" prepend-icon="mdi-earth">
           <v-divider></v-divider>
           <v-card-text>
-            <h3 class="mb-1">Default currency</h3>
+            <v-autocomplete
+              v-model="userSettingsForm.selectedCountry"
```

Comment:
this seems overkill (and not working) for the v1 of the translation mecanism :)

I'd expect a simple language selector (with only the `constants.LANGUAGE_LIST` + stored in the store)